Bottom: 152589b28a155ce91a86fb6313b8d21d488eff76
Top:    a689f0b83cb55d0addada09b6bd6a3bba6efd6a5
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-05-26 08:29:03 -0400

Refresh of mpc5200-dma

---

diff --git a/sound/soc/fsl/mpc5200_dma.h b/sound/soc/fsl/mpc5200_dma.h
index 7414c6f..2000803 100644
--- a/sound/soc/fsl/mpc5200_dma.h
+++ b/sound/soc/fsl/mpc5200_dma.h
@@ -77,18 +77,4 @@ int mpc5200_audio_dma_destroy(struct of_device *op);
 
 extern struct snd_soc_platform mpc5200_audio_dma_platform;
 
-/* whack this after Timur's patch is merged in to arch/powerpc/include/asm/delay.h */
-#define spin_event_timeout(condition, timeout, delay, rc)                   \
-{                                                                           \
-       unsigned long __loops = tb_ticks_per_usec * timeout;                \
-       unsigned long __start = get_tbl();                                  \
-       while ((rc = (condition)) && (tb_ticks_since(__start) <= __loops)) \
-               if (delay)                                                  \
-                       udelay(delay);                                      \
-               else                                                        \
-                       cpu_relax();                                        \
-}
-/* whack this after Timur's patch is merged in to arch/powerpc/include/asm/delay.h */
-
-
 #endif /* __SOUND_SOC_FSL_MPC5200_DMA_H__ */
